### PR TITLE
feat: add topic

### PR DESCRIPTION
**Pull request by yes1am**
- 新增 getTabsSquareFeed, getTabsSelectedFeed 获取圈子动态
- 更新 README.md, 新增 Development.md 便于新人参与开发

### CODE REVIEW

**Comment by sxzz** [comment]
很棒！建议把`Development.md`独立出来，重新发一个PR。并命名为 `CONTRIBUTING.md`

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ dist
 docs
 .DS_Store
 .pnpm-debug.log*
+tests/config.js
```
Comment (sxzz) [inline .gitignore]: I added `config.ts` in `tests/.gitignore` already.

File: src/api/topic.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import type { PaginationOption } from '../types/options'
+
+/**
+ * 获取圈子动态
+ * @param topicId 圈子 id
+ * @param option 选项
+ */
+export const getTabsSquareFeed = <T = Topic.GetTabsSquareFeedResponse>(
+  topicId: string,
+  option: PaginationOption = {}
```
Comment (sxzz) [inline src/api/topic.ts]: Please add a type for `PaginationOption`

**Review by sxzz — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (yes1am) [reply .gitignore]: done

File: src/api/topic.ts (hunk 2 above)
Comment (yes1am) [reply src/api/topic.ts]: done

**Review by sxzz — APPROVED** [review]

**Pull request merged**